crud-webapp/public/index.html

```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>$100k Club - CRUD Manager</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <h1>$100k Club - Comic Data Manager</h1>
+        <nav>
+            <button onclick="showSection('books')" class="nav-btn active" id="books-nav">Books</button>
+            <button onclick="showSection('records')" class="nav-btn" id="records-nav">Records</button>
+            <button onclick="showSection('pedigrees')" class="nav-btn" id="pedigrees-nav">SA Pedigrees</button>
+        </nav>
+    </header>
+
+    <main>
+        <!-- Books Section -->
+        <section id="books-section" class="data-section active">
+            <h2>Comic Books Management</h2>
+            
+            <div class="actions">
+                <button onclick="showAddBookForm()" class="btn btn-primary">Add New Book</button>
+                <input type="text" id="books-search" placeholder="Search books..." onkeyup="filterBooks()">
+            </div>
+
+            <!-- Add/Edit Book Form -->
+            <div id="book-form-container" class="form-container" style="display: none;">
```

Comment:
We need the ability to edit sales for an individual book. The records.json data is separate to the sales in each book